pkg/lib/bundle/validate.go

```diff
@@ -165,6 +193,61 @@ func (i imageValidator) ValidateBundleFormat(directory string) error {
 		validationErrors = append(validationErrors, err)
 	}
 
+	// Validate dependencies if exists
+	for _, d := range dependenciesFile.Dependencies {
+		switch d.GetType() {
+		case "olm.gvk":
+			gvkDep := &registry.GVKDependency{}
+			if d.GetValue() == "" {
```

Comment:
this can be a followup, but I think this would be more readable if it were refactored a bit:

```go
ValidatorFor(d.GetType().Validate()
```